app/api/helpers/permissions.py

```diff
@@ -145,6 +146,30 @@ def decorated_function(*args, **kwargs):
     return decorated_function
 
 
+@second_order_decorator(jwt_required)
+def get_event_id(f):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Argument name "f" doesn't conform to '[a-z_][a-z0-9_]{1,50}$' pattern](https://app.codacy.com/app/fossasia/open-event-orga-server/pullRequest?prid=4448101)